app/scenes/Collection.js

```diff
@@ -65,6 +65,15 @@ class CollectionScene extends React.Component<Props> {
   componentDidUpdate(prevProps) {
     const { id } = this.props.match.params;
 
+    if (this.collection) {
+      const { collection } = this;
+      const policy = this.props.policies.get(collection.id);
+
+      if (!policy) {
+        this.loadContent(collection.id);
+      }
+    }
```

Comment:
- check for policy presence. this is important for permissions controls too!
- we're omitting the carve-out for `collection.deletedAt` for now, but we can add special handling down the line if that's desired